src/status_im/ui/screens/chat/views.cljs

```diff
@@ -109,9 +110,48 @@
 
 (defn empty-chat-container
   []
-  [react/view style/empty-chat-container
-   [react/text {:style style/empty-chat-text}
-    (i18n/label :t/empty-chat-description)]])
+  (let [{:keys [color chat-id chat-name]}
+        @(re-frame/subscribe [:chats/current-chat])
+
+        link
+        (universal-links/generate-link :public-chat :external chat-id)
+
+        message
+        (i18n/label :t/share-public-chat-text {:link link})
+
+        public-chat-name-style
```

Comment:
if you don't inline the style directly it's convention to put it in styles.cljs next to the views

Reply:
got it!

Reply:
done!